### PR TITLE
Text Color Update

### LINKED ISSUE #277: Update text colors on utilities page
*.text-primary #3074af*
*.text-danger #E00000*

Colors/variables may be used elsewhere, please audit usage

### PR DESCRIPTION
**Pull request by kevinparkerson**
Fixes #277 by updating text colors to desired values

### CODE REVIEW

**Comment by kevinparkerson** [comment]
I scanned the whole project directory for `text-primary` and `text-danger`, only found the instances I've updated.

File: less/bootstrap-override/type.less [hunk 1]
```diff
@@ -105,7 +108,7 @@ h6, .h6 { font-size: @font-size-h6; }
 	color: #757575 !important;
 }
 .text-primary {
-	color: #2075c0 !important;
+	color: #3074af !important;
```
Comment (kevinparkerson) [inline less/bootstrap-override/type.less]: I noticed there were `!important` declarations here... should we be doing this??? I added the `.text-danger` declaration above to account for the updated color, but did not add `!important` onto it. It still overrides the bootstrap style as expected. Unless there are solid reasons for doing so, we should avoid `!important` declarations in our LESS.

**Comment by swilliamset** [comment]
Create vars for the color(s) and have the class consume the vars

**Comment by kevinparkerson** [comment]
@interactivellama, @jmgekeler hey guys, there were some `!important` declarations on the bootstrap text color overrides... I've gotten rid of those, and it still appears to override the underlying bootstrap style. Is there a solid reason why those were there? If not this can go through. We should create another issue to try and get rid of as many `!important` declarations as possible as it breaks the cascading nature of CSS, making it difficult for developers to override them if they need to later on.

**Comment by kevinparkerson** [comment]
@interactivellama said the `!important` declarations did not have a major intention and were pulled in from elsewhere. It should be safe to remove them

**Pull request merged**